docs/tutorials/vis_cpu_tutorial.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    flux = np.random.random(nsource) * 4
</pre><p>Here there's no need to create the <code>sources</code>  array. Just keep <code>ra</code>,  <code>dec</code>  and <code>flux</code>  as separate arrays.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/vis_cpu/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I have modified the notebook as suggested.